README.md

```diff
@@ -1,7 +1,8 @@
-Kymatio: wavelet scattering in PyTorch
+Kymatio: Wavelet scattering in PyTorch
 ======================================
 
 Kymatio is an implementation of the wavelet scattering transform in the Python programming language, suitable for large-scale numerical experiments in signal processing and machine learning.
+Scattering transforms are translation-invariant signal representations implemented as convolutional networks whose filters are not learned but fixed wavelet filters.
```

Comment:
hmm does the last part read right to you? "are not [adjective] but [noun-related-thing]". I'd go with "not learned, but fixed (as wavelet filters)" or something

Reply:
You're right. I'll change it.